### PR TITLE
Add Best Practices Section

### PR DESCRIPTION
**Pull request by RoAnnC1**
@sarraloew 

Take a look and see what you think.

### CODE REVIEW

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 1]
```diff
@@ -241,19 +252,21 @@ Content-Type: application/xml
...
 ```http
 GET /api/travel/trip/v1.1/?createdAfterDate=2012%2F02%2F01&includeMetadata=true&ItemsPerPage=2&Page=1 HTTP/1.1
 Host: www.concursolutions.com
 Authorization: OAuth {access token}
 ```
 
 #### Response
-The response returns a ConnectResponse parent element which contains a MetaData element with paging information and a Data element with an ItineraryInfoList child element.
+
+The response returns a `ConnectResponse`` parent element which contains a MetaData` element with paging information and a `Data` element with an `ItineraryInfoList` child element.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: The code formatting is a weird here.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 2]
```diff
@@ -300,51 +313,51 @@ Content-Type: application/xml
...
 
-  GET /travel/trip/v1.1/trip_ID?[systemFormat=system_format|&userid_type=login|&user_id=login_ID]
+```GET /travel/trip/v1.1/trip_ID?[systemFormat=system_format|&userid_type=login|&user_id=login_ID]```
 
 ## Path Parameters
 
 | Parameter Name | Data Type |Description
 | --------- | --------- | -------
-| trip_ID (required) | string  |  The identifier for the desired trip. This identifier is returned as the value of the id element when getting trip summaries. For example, if the returned value of the id element is I2uwiJJw8r7Owl3IWlSie9WIelxhAhwiL, then the URI for the request is `/travel/trip/v1.1/I2uwiJJw8r7Owl3IWlSie9WIelxhAhwi`
+| trip_ID (required) | string  |  The identifier for the desired trip. This identifier is returned as the value of the ID element when getting trip summaries. For example, if the returned value of the ID element is I2uwiJJw8r7Owl3IWlSie9WIelxhAhwiL, then the URI for the request is `/travel/trip/v1.1/I2uwiJJw8r7Owl3IWlSie9WIelxhAhwi`.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Let's use the standard bold Required in the description

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 3]
```diff
@@ -300,51 +313,51 @@ Content-Type: application/xml
...
 
 
 ## Query Parameters
 
 | Parameter Name |Data Type |Description
 | --------- | --------- | -------
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown:343]: And let's move "optional" to the description also.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 4]
```diff
@@ -353,44 +366,43 @@ The response returns subset of the elements described in the following tables de
 | id	|string	|Trip ID URI with encrypted ID.
 |ItinLocator | string	|The itinerary locator. This element is now deprecated and only supported for backward compatibility.
 |ClientLocator	|string	|The XSD does not have a description for this element. Please provide one.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown:368]: Whoops. ClientLocator has reviewer notes in it again. Let's remove the existing description and send a query to the Travel PM.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 5]
```diff
@@ -353,44 +366,43 @@ The response returns subset of the elements described in the following tables de
...
-|Passengers	|array	|Contains a Passenger child element for each included passenger. For the descriptions of these elements, see Passengers Elements
-|RecordLocator	|string	|The unique identifier for a booking. This is often six alphanumeric characters but can have other formats depending on the booking source.
+|Segments	|array	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
+|Passengers	|array	|Contains a `Passenger` child element for each included passenger. For the descriptions of these elements, see Passengers Elements.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: This looks like it used to be a link. There is no table for Passengers elements, but there is a more detailed description for Passengers in the Create a New Trip section, so let's link to that.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 6]
```diff
@@ -652,38 +663,38 @@ This endpoint is used for creating a new trip. To create a new trip, the specifi
 
 | Parameter Name |Data Type |Description
 | --------- | --------- | -------
-| userid_type (optional) | string	| The type of user identification to use. Possible value is: `login_id`
-| userid_value (optional) | string	| The value for the user identification type. Currently the only available type is `login_id` so the value is the login credentials. This parameter must be provided in conjunction with the `userid_type` parameter. The `userid_type` and `userid_value` parameters can only be used if the user account associated with the OAuth 2.0 access token is associated with a Concur account with one of these roles: Web Services Administrator for Professional or Can Administer for Standard. The format for the request URI using the `userid_type` and `userid_value` query parameters is `/travel/trip/v1.1/trip_ID?userid_type=login&userid_value=login_ID`
+| userid_type (optional) | string	| The type of user identification to use. Possible value is: `login_id`.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Let's move "optional" to the description

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 7]
```diff
@@ -652,38 +663,38 @@ This endpoint is used for creating a new trip. To create a new trip, the specifi
...
+Where `access_token` is the OAuth 2.0 access token of the user whose trip you want to create or update. If you want to access company-wide itinerary information, the user account associated with the OAuth 2.0 access token must have an SAP Concur account with one of these roles: Web Services Administrator for Professional or Can Administer for Standard.
 
 ### Request
 
 | Element Name | Required or Optional | TripLink | Data Type | Description
 | --------- | --------- | ------- | -------- | ---------
-| Itinerary | required | Y | ItineraryType	| The root element for a trip. For this endpoint, it contains the following elements: ClientLocator, ItinSourceName, TripName, Comments, StartDateLocal, EndDateLocal, BookedByFirstName, BookedByLastName, Bookings.
+| Itinerary | required | Y | ItineraryType	| The root element for a trip. For this endpoint, it contains the following elements: `ClientLocator`, `ItinSourceName`, `TripName`, `Comments`, `StartDateLocal`, `EndDateLocal`, `BookedByFirstName``, BookedByLastName`, `Bookings`.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: BookedByLastName code backticks are weird

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 8]
```diff
@@ -652,38 +663,38 @@ This endpoint is used for creating a new trip. To create a new trip, the specifi
...
+| Bookings | optional | Y | array	| A parent element that contains a `Booking` child element for each booking associated with this itinerary.
+| Booking | optional | Y | array	| A child element of the `Bookings` element which in turn contains the following child elements: `Segments`, `Passenger`s, `RecordLocator`, `BookingSource`, `DateModifiedUtc`, `DateBookedLocal`, `ItinSourceName`, and `PassengerCount`.
+| Segments | optional	 | Y | array	| List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segment`s element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. For TripLink suppliers, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Code backticks on first Segments are not quite right. But it seems to render ok?

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 9]
```diff
@@ -979,7 +991,8 @@ Content-Type: application/xml
 </Itinerary>
 ```
 
-### Example 2: TripLink supplier creates a trip
+### Example 2: TripLink Supplier Creates a Trip
+
 This example shows how a TripLink supplier create a trip.
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: creates

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown [hunk 10]
```diff
@@ -1262,17 +1276,15 @@ Content-Type: application/xml
 #### Response
 The response is the same as in Example 1.
 
+## <a name="postut"></a>[Update a Trip](/api-reference/travel/itinerary-tmc-thirdparty/index.html#postdetails)
```
Comment (sarraloew) [inline src/api-reference/travel/itinerary/trip/trip-resource.markdown]: This looks like an old link from a previous site version, please remove

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 2 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Fixed, note there is a table that has a column for optional v. required and I left that as is.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 4 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown:368]: You sent the query the last time. I don't have the contact info.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 5 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Done, tweaked the description slightly.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 6 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Done

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 7 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 8 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Fixed

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 9 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: ARGH, reading is a thing.

File: src/api-reference/travel/itinerary/trip/trip-resource.markdown (hunk 10 above)
Comment (RoAnnC1) [reply src/api-reference/travel/itinerary/trip/trip-resource.markdown]: Done

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Pull request merged**